js/app/modules/contentGroup/mediaLightbox.js

```diff
@@ -57,6 +57,8 @@ const MediaLightbox = new Module.Class({
     },
 
     _on_close: function () {
+        this.lightbox_widget.destroy();
```

Comment:
If you're keeping a reference to a widget in your JS code, you *must* connect to the `destroy` signal and release your reference. This should always happen.